image/src/shared/interfaces/request-with-user.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface RequestWithUser extends Request {
+  // TODO: Once a user is specifed update this
+  user: any;
```

Comment:
Any?

Reply:
This is temporary until definition of the user is clear. 
This prevents having to maintain multiple type definitions for a longer time.